index.bs

```diff
@@ -2272,34 +2298,113 @@ stream data with any data not consumed being discarded. However,
 immediate signaling is not required. Also, if stream data
 is completely received but has not yet been read by the
 application, the RESET_STREAM signal can be suppressed.
-  
+
   <table class="data">
     <colgroup class="header"><col></colgroup>
     <colgroup><col></colgroup>
     <thead>
       <tr>
-        <th>WebTransport Protocol Action</th>
+        <th>HTTP/3 Protocol Action</th>
         <th>API Effect</th>
       </tr>
     </thead>
     <tbody>
       <tr>
-        <td>received [=session-signal/DRAIN_WEBTRANSPORT_SESSION=]</td>
+        <td>received [=session-signal/GOAWAY=]</td>
         <td>await wt.{{WebTransport/draining}}</td>
       </tr>
     </tbody>
   </table>
-  
+
+If the [=underlying connection=] is using HTTP/2, the following protocol behaviors
+from [[!WEB-TRANSPORT-HTTP2]] apply.
+
+
```

Comment:
Language about how to interpret streamErrorCode in HTTP/2 seems to be missing. Is that described anywhere we can link to, e.g. https://datatracker.ietf.org/doc/html/draft-ietf-webtrans-http2/#section-5.2?

Reply:
IIUC, we don't need to convert the streamErrorCode in HTTP/3 and can just use the error as is, so I added a note to clarify that.